<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Example of output from SpectroMine for TMT-6plex experiments. — raw.mine • MSstatsTMTdev</title>


<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha256-916EbMg70RQy9LHiGkXzG8hSg9EdNy97GazNG/aiY1w=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.7.1/css/all.min.css" integrity="sha256-nAmazAk6vS34Xqo0BSrTb+abbtFlgsFK7NKSi6o7Y78=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.7.1/css/v4-shims.min.css" integrity="sha256-6qHlizsOWFskGlwVOKuns+D1nB6ssZrHQrNj1wGplHc=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.9.4/headroom.min.js" integrity="sha256-DJFC1kqIhelURkuza0AvYal5RxMtpzLjFhsnVIeuk+U=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.9.4/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>




<meta property="og:title" content="Example of output from SpectroMine for TMT-6plex experiments. — raw.mine" />
<meta property="og:description" content="Example of SpectroMine PSM sheet.
It is the output of SpectroMine and the input for SpectroMinetoMSstatsTMTFormat function,
with annotation file.
Annotation file should be made by users.
It includes peak intensities for 10 proteins among 12 MS runs with TMT-6plex.
The important variables are as follows:" />
<meta name="twitter:card" content="summary" />




<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body>
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">MSstatsTMTdev</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.9.3</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fas fa fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="../articles/MSstatsTMT.html">MSstatsTMT : A package for protein significance analysis in shotgun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Example of output from SpectroMine for TMT-6plex experiments.</h1>
    
    <div class="hidden name"><code>raw.mine.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>Example of SpectroMine PSM sheet.
It is the output of SpectroMine and the input for SpectroMinetoMSstatsTMTFormat function,
with annotation file.
Annotation file should be made by users.
It includes peak intensities for 10 proteins among 12 MS runs with TMT-6plex.
The important variables are as follows:</p>
    </div>

    <pre class="usage"><span class='no'>raw.mine</span></pre>


    <h2 class="hasAnchor" id="format"><a class="anchor" href="#format"></a>Format</h2>

    <p>A data frame with 170 rows and 28 variables.</p>
    <h2 class="hasAnchor" id="details"><a class="anchor" href="#details"></a>Details</h2>

    
<ul>
<li><p>PG.ProteinAccessions</p></li>
<li><p>P.MoleculeID</p></li>
<li><p>PP.Charge</p></li>
<li><p>R.FileName</p></li>
<li><p>PG.QValue</p></li>
<li><p>PSM.Qvalue</p></li>
<li><p>Channels : PSM.TMT6_126..Raw., ..., PSM.TMT6_131..Raw.</p></li>
</ul>


    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'><span class='fu'><a href='https://rdrr.io/r/utils/head.html'>head</a></span>(<span class='no'>raw.mine</span>)</div><div class='output co'>#&gt;   R.MS3.Used R.BlockName R.QuantificationMethod
#&gt; 1       True    All Runs               TMT6Plex
#&gt; 2       True    All Runs               TMT6Plex
#&gt; 3       True    All Runs               TMT6Plex
#&gt; 4       True    All Runs               TMT6Plex
#&gt; 5       True    All Runs               TMT6Plex
#&gt; 6       True    All Runs               TMT6Plex
#&gt;                                              R.FileName PG.Genes PG.Organisms
#&gt; 1 ch_19Jan2017_SM-1-1_Sp-6-2_CID-OT-MS3-Short_HpH_1.raw    EGLN1 Homo sapiens
#&gt; 2 ch_19Jan2017_SM-1-1_Sp-6-2_CID-OT-MS3-Short_HpH_1.raw   SEPT11 Homo sapiens
#&gt; 3 ch_19Jan2017_SM-1-1_Sp-6-2_CID-OT-MS3-Short_HpH_1.raw   SEPT11 Homo sapiens
#&gt; 4 ch_19Jan2017_SM-1-1_Sp-6-2_CID-OT-MS3-Short_HpH_1.raw   SEPT11 Homo sapiens
#&gt; 5 ch_19Jan2017_SM-1-1_Sp-6-2_CID-OT-MS3-Short_HpH_1.raw   SEPT11 Homo sapiens
#&gt; 6 ch_19Jan2017_SM-1-1_Sp-6-2_CID-OT-MS3-Short_HpH_1.raw   NAP1L4 Homo sapiens
#&gt;   PG.ProteinAccessions               PG.ProteinDescriptions PG.ProteinNames
#&gt; 1               Q9GZT9                   Egl nine homolog 1     EGLN1_HUMAN
#&gt; 2               Q9NVA2                            Septin-11     SEP11_HUMAN
#&gt; 3               Q9NVA2                            Septin-11     SEP11_HUMAN
#&gt; 4               Q9NVA2                            Septin-11     SEP11_HUMAN
#&gt; 5               Q9NVA2                            Septin-11     SEP11_HUMAN
#&gt; 6               Q99733 Nucleosome assembly protein 1-like 4     NP1L4_HUMAN
#&gt;   PG.UniprotIds PG.Coverage PG.QValue PEP.IsProteinGroupSpecific
#&gt; 1        Q9GZT9        5.6%         0                       TRUE
#&gt; 2        Q9NVA2        9.1%         0                       TRUE
#&gt; 3        Q9NVA2        9.1%         0                      FALSE
#&gt; 4        Q9NVA2        9.1%         0                       TRUE
#&gt; 5        Q9NVA2        9.1%         0                       TRUE
#&gt; 6        Q99733        5.1%         0                       TRUE
#&gt;   PEP.IsProteotypic     PEP.StrippedSequence   PEP.QValue
#&gt; 1              TRUE AAAGGQGSAVAAEAEPGKEEPPAR 0.0000000000
#&gt; 2              TRUE             KELEEEVNNFQK 0.0001986492
#&gt; 3             FALSE                 SLDLVTMK 0.0000000000
#&gt; 4              TRUE      AAAQLLQSQAQQSGAQQTK 0.0000000000
#&gt; 5              TRUE      AAAQLLQSQAQQSGAQQTK 0.0000000000
#&gt; 6              TRUE                 VLAALQER 0.0024595109
#&gt;   PEP.IsUsedForQuantification PP.Charge
#&gt; 1                        True         3
#&gt; 2                        True         3
#&gt; 3                        True         2
#&gt; 4                        True         3
#&gt; 5                        True         2
#&gt; 6                        True         2
#&gt;                                    P.MoleculeID PSM.TMT6_126..Raw.
#&gt; 1 _[TMT_Nter]AAAGGQGSAVAAEAEPGK[TMT_Lys]EEPPAR_           382.1107
#&gt; 2    _[TMT_Nter]K[TMT_Lys]ELEEEVNNFQK[TMT_Lys]_         33554.1900
#&gt; 3                 _[TMT_Nter]SLDLVTMK[TMT_Lys]_         44713.6300
#&gt; 4      _[TMT_Nter]AAAQLLQSQAQQSGAQQTK[TMT_Lys]_         20877.8700
#&gt; 5      _[TMT_Nter]AAAQLLQSQAQQSGAQQTK[TMT_Lys]_           506.1669
#&gt; 6                          _[TMT_Nter]VLAALQER_         17143.5200
#&gt;   PSM.TMT6_127..Raw. PSM.TMT6_128..Raw. PSM.TMT6_129..Raw. PSM.TMT6_130..Raw.
#&gt; 1           392.9243           477.7064           989.6951           695.7537
#&gt; 2         33671.5400         40525.3900         43739.8600         40697.4400
#&gt; 3         51052.8400         58457.6400         47213.1300         58004.6700
#&gt; 4         19930.0100         24963.5400         29021.8300         19510.8600
#&gt; 5          1019.8020           543.2091           555.3629           501.8123
#&gt; 6          4284.8930         26957.7100         15610.6900         21208.9500
#&gt;   PSM.TMT6_131..Raw. PSM.IsUsedForQuantification PSM.NrOfMatchedChannelIons
#&gt; 1           107.6282                        True                          6
#&gt; 2         17299.1300                        True                          6
#&gt; 3         23622.9500                        True                          6
#&gt; 4         15952.0600                        True                          6
#&gt; 5           268.5246                        True                          6
#&gt; 6          7701.3510                        True                          6
#&gt;     PSM.Qvalue
#&gt; 1 0.000000e+00
#&gt; 2 2.146015e-04
#&gt; 3 0.000000e+00
#&gt; 4 5.613247e-05
#&gt; 5 0.000000e+00
#&gt; 6 1.951193e-03</div><div class='input'>
</div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
    <h2>Contents</h2>
    <ul class="nav nav-pills nav-stacked">
      <li><a href="#format">Format</a></li>
      <li><a href="#details">Details</a></li>
      <li><a href="#examples">Examples</a></li>
    </ul>

  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by Ting Huang, Meena Choi, Mateusz Staniak, Sicheng Hao, Olga Vitek.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.4.1.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


